lib/tasks/hdm.rake

```diff
@@ -22,7 +24,7 @@ namespace :hdm do
   task :load, %i[file create_user account_email account_password] => :environment do |_t, args|
     bundle_json = File.open(args.file, 'r:UTF-8').read
     json_format = JSON.parse(bundle_json)
-    user_indx = 0
+    user = ''
     if args.create_user == 'y'
       patient_entry = json_format['entry'].find \
```

Comment:
I didn't notice this before but you should also move the functionality of creating  the user from a fhir bundle into the  library file as well